Source/Appender.swift

```diff
@@ -0,0 +1,7 @@
+public protocol Appender : Stream {
```

Comment:
In time I will learn to use the magnetic force of the colons to my advantage. For now I will need occasional reminders :)